### PR TITLE
Implement lynx browser for Issue #1

### PR DESCRIPTION
**Pull request by gsornsen**
Let me know your thoughts on this, I'm open to other implementations.

There appears to be a conflict(s) that require resolution before merging.

### CODE REVIEW

**Comment by gsornsen** [comment]
Conflicts resolved.

File: README.md [hunk 1]
```diff
@@ -30,11 +30,33 @@ or libraries are required.
...
+the browser, search engine, and terminal (if using Lynx browser).
 
-Supported browsers are Google Chrome, Firefox, Brave, and Chromium. Supported search engines are
-Google and DuckDuckGo. Just open the script and edit the two variables
-`SEARCH_ENGINE` and `BROWSER` at the geginning of the script.
+Supported browsers are Google Chrome, Firefox, Brave, Chromium, and Lynx.
+
+Supported search engines are Google and DuckDuckGo.
+
+If using Lynx, the supported browsers are `st` and `gnome-terminal`
```
Comment (gsornsen) [inline README.md]:
```suggestion
If using Lynx, the supported terminals are `st` and `gnome-terminal`
```

**Comment by pdonadeo** [comment]
@gsornsen I decided to discard your PR in favour of a more simple configuration and removing another "if".

Do you mind to submit another PR with your version of the README, so that you remain in the history?

**Comment by gsornsen** [comment]
> @gsornsen I decided to discard your PR in favour of a more simple configuration and removing another "if".
> 
> Do you mind to submit another PR with your version of the README, so that you remain in the history?

Sure thing and No problem

**Pull request closed without merging**